src/app/home/shared/forms/income-form/income-form.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="container" @fade>
   <div class="main-icon-container">
-    <i class="fas fa-money-bill-wave main-icon"></i>
+    <img src="assets/images/main-icon.svg" alt="image" class="main-icon">
```

Comment:
The purpose of `alt` is to describe what image is that, for better accessibility. I'm afraid, 'image' might be a little bit generic. Just a little bit. Generic. A little.